### DISCUSSION TITLE
Version 2.9 Expected Release Date

### DISCUSSION THREAD

**Discussion by zfields** [stm32duino/Arduino_Core_STM32#2554] [General]
Do you have a target for releasing stm32duino v2.9?

There is a reasonably important update to the Blues Cygnet, and we would like to understand your timeline so we can plan accordingly.

Thanks,
Zak

**Comment by fpistm** [comment]
Hi @zfields 
Next week I think.

**Reply by fpistm** [reply]
@zfields 
what is you deadline for this release ?

**Reply by zfields** [reply]
I think we were shooting for Dec. 4th as our early date, but it may be Dec. 15th now.

**Reply by fpistm** [reply]
Ok . I plan to release next week. So this will not be a problem.

**Reply by fpistm** [reply]
2.9.0 released.
